docs/authentication/saml/azure.mdx

```diff
@@ -17,77 +17,83 @@ description: Learn how to allow users to sign into your Clerk app with their Mic
+  1. Navigate to the [**Clerk Dashboard**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
   1. In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**.
-  1. Select the **Add connection** button, and select **For specific domains**.
-  1. Select **Microsoft Entra ID (Formerly AD)** as the identity provider.
-  1. Add the **Name** of the connection. This is the name that will be displayed in the sign-in form.
-  1. Add the **Specific Domain** that you want to allow this connection for. This is the domain of the users you want to allow to sign in to your application.
+  1. Select **Add connection**, and select **For specific domains**.
+  1. In the modal that appears, select **Microsoft Entra ID (Formerly AD)** as the identity provider.
+  1. Add the **Name** of the connection. This is the name that will be displayed on the sign-in form.
+  1. Enter the **Specific Domain** that you want to allow this connection for. This is the domain of the users you want to allow to sign in to your application.
   1. Select **Add connection**. You will be redirected to the connection's configuration page.
   1. Find the **Service Provider Configuration** section.
-  1. Save the **Reply URL (Assertion Consumer Service URL)** and the **Identifier (Entity ID)** values somewhere secure. You'll need these in the [Configure your service provider](#configure-your-service-provider) step.
-  1. Leave this page open.
+  1. Save the **Identifier (Entity ID)** and **Reply URL (Assertion Consumer Service URL)** values somewhere secure. You'll need these in the [Configure your service provider](#configure-your-service-provider) step. Leave this page open.
 
   ### Create a new enterprise application in Microsoft
 
   To create a new enterprise application in Microsoft:
 
-  1. Navigate to the [Microsoft Portal](https://azure.microsoft.com/en-us/get-started/azure-portal) and sign in.
-  1. Under the **Azure Services** section, find and select **Enterprise Applications**. You may have to go to the [**All services**](https://portal.azure.com/#allservices) page and then scroll down to the **Identity** section to find it.
-  1. Select the **New Application** button.
-  1. Select the **Create your own application** button.
-  1. Fill in the **Name**, ensure **Integrate any other application you don't find in the gallery (Non-gallery)** is selected, and select the **Create** button.
+  1. In a separate page, navigate to the [**Microsoft Azure portal**](https://azure.microsoft.com/en-us/get-started/azure-portal) and sign in.
+  1. Under the **Azure Services** section, find and select **Enterprise applications**. You may have to go to the [**All services**](https://portal.azure.com/#allservices) page and then scroll down to the **Identity** section to find it.
+  1. After being redirected to the **All applications** page, select **New application**.
```

Comment:
There's a whole step telling them what page they are going to, so I don't think this first part is necessary. 

However, in this step, we can tell the user where they are being redirected to after their action, instead of having this info in the next step. It feels more intuitive than "Select new application. In the Browse Microsoft Entra Gallery page, select create your own application" because those instructions sound like you, as the user, need to now navigate to the browse microsoft entra gallery page.

```suggestion
  1. Select **New application**. You'll be redirected to the **Browse Microsoft Entra Gallery** page.
```